test/e2e/tests/add-custom-network.spec.js

```diff
@@ -458,7 +458,7 @@ describe('Custom network', function () {
           });
           // verify network switched
           const networkDisplayed = await driver.findElement({
-            tag: 'p',
+            tag: 'span',
```

Comment:
Changed the wrapping html element of the text from a `p` to `span` tag in the `PickerNetwork`